package/kedro_viz/models/flowchart/pipeline.py

```diff
@@ -0,0 +1,7 @@
+"""`kedro_viz.models.flowchart.pipelines` represent Kedro pipelines in a viz graph."""
+
+from .model_utils import NamedEntity
+
+
+class RegisteredPipeline(NamedEntity):
+    """Represent a registered pipeline in a Kedro project."""
```

Comment:
I am not sure about this file but I know it aligns the way we have broken things down. Let's keep it as is for now.

Reply:
I think I liked your original way of doing it, calling it NamedEntity.py , and having all three classes in there (NamedEntity, RegisteredPipeline and Tags). I know NamedEntity is not clear by just reading the filename, but with the definition it gives clarity. Should we go back to before ? wdyt ?